bpdm-bridge-dummy/src/main/kotlin/com/catenax/bpdm/bridge/dummy/service/PoolUpdateService.kt

```diff
@@ -53,16 +54,30 @@ class PoolUpdateService(
     }
 
     fun updateLegalEntitiesInPool(entriesToUpdate: Collection<GateLegalEntityInfo>): LegalEntityPartnerUpdateResponseWrapper {
-        val updateRequests = entriesToUpdate.map {
-            LegalEntityPartnerUpdateRequest(
-                legalEntity = gateToPoolLegalEntity(it.legalEntity),
-                legalAddress = gateToPoolLogisticAddress(it.legalAddress.address),
-                bpnl = it.bpn!!
-            )
+        val errorListException = mutableListOf<ErrorInfo<LegalEntityUpdateError>>()
+        val updateRequests = mutableListOf<LegalEntityPartnerUpdateRequest>()
+
+        entriesToUpdate.forEach {
+            try {
+                updateRequests.add(
+                    LegalEntityPartnerUpdateRequest(
+                        legalEntity = gateToPoolLegalEntity(it.legalEntity),
+                        legalAddress = gateToPoolLogisticAddress(it.legalAddress.address),
+                        bpnl = it.bpn!!
+                    )
+                )
+            } catch (e: BpdmNullMappingException) {
+                logger.info { "Error processing ${it.externalId}" }
+                val errorMapped = ErrorInfo(LegalEntityUpdateError.LegalEntityDuplicateIdentifier, "Error on mapping '${it.legalEntity}' message ${e.message}", it.bpn)
```

Comment:
When I test locally I run into another error here because the error message is too long for the sharing state as this is a 255 VARCHAR.